backend/src/schema.ts

```diff
@@ -107,36 +107,45 @@ export const lists: Lists = {
         }
     }),
 
-    Medicine: list({
+    Therapeutic: list({
         access: allowAll,
         fields: {
             name: text({ validation: { isRequired: true } }),
-            study: relationship({ ref: "Study.medicine" }),
-            medicineCaptures: relationship({
-                ref: "MedicineCapture.medicine",
-                many: true
-            })
+            study: relationship({ ref: "Study.therapeutic" })
         }
     }),
 
-    MedicineAssignment: list({
+    TherapeuticAssignment: list({
         access: allowAll,
         fields: {
-            user: relationship({ ref: "User.medicineAssignments" }),
-            study: relationship({ ref: "Study.medicineAssignment" }),
+            user: relationship({ ref: "User.therapeuticAssignments" }),
+            study: relationship({ ref: "Study.therapeuticAssignment" }),
             quantity: integer({ validation: { min: 0, isRequired: true } }),
-            directions: text({ validation: { isRequired: true } }),
+            therapeuticAssignmentSteps: relationship({ ref: "TherapeuticAssignmentSteps.therapeuticAssignment", many: true }),
+            therapeuticCaptures: relationship({
+                ref: "TherapeuticCapture.therapeuticAssignment",
+                many: true
+            }),
             startDate: timestamp({ validation: { isRequired: true } }),
             endDate: timestamp({ validation: { isRequired: true } })
         }
     }),
 
-    MedicineCapture: list({
+    TherapeuticAssignmentStep: list({
+        access: allowAll,
+        fields: {
+            therapeuticAssignment: relationship({ ref: "TherapeuticAssignment.therapeuticAssignmentSteps" }),
+            index: integer({ validation: { isRequired: true, min: 0 } }),
+            direction: text({ validation: { isRequired: true } })
+        }
+    }),
+
+    TherapeuticCapture: list({
         access: allowAll,
         fields: {
-            user: relationship({ ref: "User.medicineCaptures" }),
+            user: relationship({ ref: "User.therapeuticCaptures" }),
             time: timestamp({ validation: { isRequired: true } }),
-            medicine: relationship({ ref: "Medicine.medicineCaptures" })
+            therapeuticAssignment: relationship({ ref: "TherapeuticAssignment.therapeuticCaptures" })
         }
```

Comment:
Since we now have therapeutic assignment steps, we'll now need therapeutic capture steps to capture each step of the therapeutic assignment